modules/ROOT/pages/migrate.adoc

```diff
@@ -12,7 +12,7 @@ Migration to Exchange 2 differs by asset type:
 |Asset Type |Migration Approach
 |Example, Template |Anypoint Studio > Publish to Exchange.
```

Comment:
This was incorrect from the source -- should be "Example, Template |Maven - see xref:to-publish-assets-maven.adoc[Publish and Deploy Exchange Assets Using Maven]"